appveyor.yml

```diff
@@ -1,11 +1,65 @@
 image: Visual Studio 2015
 
+environment:
+  NUGET_API_KEY:
+    secure: Rh/UHYn7HGj8a7LFWo9C1rSA0sHSQNV3wIqj9bvwRR8yLmCXu9JmbNkmkctk2cBn
```

Comment:
Yes, let's wait for the @ecampidoglio reply. However, it seems that @adamchester is totally absent, so I'm not sure how much it could take to wait for his review... 

@adamchester Are you here? :)